src/main/java/com/mewna/catnip/internal/CatnipImpl.java

```diff
@@ -131,6 +139,85 @@ public boolean isUnavailable(@Nonnull final String guildId) {
         return unavailableGuilds.contains(guildId);
     }
     
+    @Override
+    public void openVoiceConnection(@Nonnull final String guildId, @Nonnull final String channelId) {
+        eventBus().send(CatnipShard.websocketMessageQueueAddress(shardIdFor(guildId)),
```

Comment:
Shards have a utility method for crafting websocket payloads; should just expose that and use it here

Reply:
@natanbc fix when reeeee